### ISSUE TITLE
no response in bot after sending auth url 

### ISSUE THREAD

**Issue by arvs47** [out386/aria-telegram-mirror-bot#4]
hello i configured this script in my server but finally i got some warning message like
node telegram bot api got deprecated and need to enable manually with this link  
https://github.com/yagop/node-telegram-bot-api/issues/319.

is it possible to run this script in windows 10 ,with node , 
if you can provide some tutorials or videos shows the configuration part would be helpful .
thank you

**Comment by out386** [comment]
That warning is normal, the bot will work anyway.
Windows, I am not sure.
Are you on windows? Nothing appears on the terminal if you send it a message?

**Comment by arvs47** [comment]
I hosted in google  cloud compute engine , in compute engine the site address is IP address , and in the auth consent json have local Host , I modified it with a site address using cloudflare by resolving the IP into URL , but in bot nothing appears ,

**Comment by arvs47** [comment]
If you do a screencast it will be nice

**Comment by out386** [comment]
Sounds like you got the wrong auth json. Read step 6 of Installation again, please. Also, I think you didn't set the bot token in .constants.js

**Comment by arvs47** [comment]
Yeah  , it seems I created new project in compute engine but I cannot create outh screen with the same project in google apis ,, I will try with an existing project , what will be the outh screen URLs is it the site address or local Host ,,

Any way I will try it once again , and let you know also it's a great project , thanks for  creating such amazing one ,,

**Comment by out386** [comment]
What URL? You need this:
https://ibb.co/ehdNMV

**Comment by arvs47** [comment]
Inside that json , there is local Host ,I configured again and when I send the readme file , I got response as you cannot use this bot here

**Comment by out386** [comment]
1) Leave that localhost as it is.
2) Configure SUDO_USERS or AUTHORIZED_CHATS in .constants.js. How to do that is in the README.

**Comment by arvs47** [comment]
so i added sudo users as my user id which is more than 3 digit like 9 digit and in authorized chat i added the chat id which is same as above more than 3 characters , how  i supposed to add this ,

**Comment by out386** [comment]
It will be like `SUDO_USERS: [123456789] ,`
If that's what it is, check your user ID, you probably mistyped it.

**Comment by arvs47** [comment]
i got response as download started and download error 

this is the console log : a2C: Websocket opened
start f7303b654f7de822
download:https://raw.githubusercontent.com/out386/aria-telegram-mirror-bot/master/README.md gid:f7303b654f7de822
error f7303b654f7de822
cleanupDownload: deleting

**Comment by arvs47** [comment]
ok i will try that .
how do i stop this ; do i need to turn off vps to stop the bot , it seems i cannot enter commands in this screen download:https://raw.githubusercontent.com/out386/aria-telegram-mirror-bot/master/README.md gid:f7303b654f7de822
error f7303b654f7de822
cleanupDownload: deleting

**Comment by out386** [comment]
`Ctrl+C`

**Comment by akhilnarang** [comment]
`*_*`

**Comment by arvs47** [comment]
ok i added the tmp folder for download and it worked ,, so far it is working great i tried with magnet link but doesn't  started yet 

0B / 0B (0%) of Metadata downloaded at 0Bps 


i have a question . this bot can be used as a  one like @getpubliclink  right , ,

**Comment by out386** [comment]
1) Too few seeds. 
2) No, this doesn't mirror files from Telegram.

**Comment by out386** [comment]
Feel free to make suggestions by opening new issues.
